eva/optimizer/operators.py

```diff
@@ -456,10 +457,13 @@ class LogicalLoadData(Operator):
     """
 
     def __init__(self, table_metainfo: DataFrameMetadata,
-                 path: Path, children=None):
+                 path: Path, file_format: FileFormatType, 
+                 file_options: dict, children=None):
         super().__init__(OperatorType.LOGICALLOADDATA, children=children)
         self._table_metainfo = table_metainfo
         self._path = path
+        self._file_format = file_format
+        self._file_options = file_options
```

Comment:
What is `file_option`? And how it is used? I don't see it in the executor. Is that resevered for future use? If so, what is the example use case?

Reply:
Yes, the design assumes file_format is one example key in file options. I followed the design used by Postgres. Other examples can be DELIMITER 'delimiter_character', HEADER [ boolean ] etc.

Reply:
@Anirudh58 we should consider only passing the file_options as a dictionary. Thanks!

Reply:
Ok got it so file_format should now be a key in file_options